project/Build.scala

```diff
@@ -63,9 +63,9 @@ object DottyJSPlugin extends AutoPlugin {
 }
 
 object Build {
-  val referenceVersion = "3.0.0"
+  val referenceVersion = "3.0.1-RC1"
```

Comment:
```suggestion
  val referenceVersion = "3.0.0"
```

Reply:
Why this change? Isn't it better to use the latest published compiler to compile Dotty?